docs/cli-help.txt

```diff
@@ -272,10 +274,15 @@ renew:
                         the shell variable $RENEWED_LINEAGE will point to the
                         config live subdirectory (for example,
                         "/etc/letsencrypt/live/example.com") containing the
-                        new certs and keys; the shell variable
+                        new certificates and keys; the shell variable
                         $RENEWED_DOMAINS will contain a space-delimited list
+<<<<<<< HEAD
```

Comment:
Looks like the merge here wasn't resolved quite right. Since this file is normally only updated during the release which prevents new flags finding their way to the website before they're available for people to use them, I think the easiest approach is to just undo any changes to this file and they'll be made automatically during the next release.